config/initializers/sidekiq.rb

```diff
@@ -1,27 +1,6 @@
 file = File.join(Rails.root, 'config', "sidekiq-#{Rails.env}.yml")
 file = File.join(Rails.root, 'config', 'sidekiq.yml') unless File.exist?(file)
 require File.join(Rails.root, 'lib', 'middleware_sidekiq_server_retry')
-require 'airbrake/sidekiq'
-
-Sidekiq::Logging.logger = Rails.logger.child
-
-Airbrake.add_filter(Airbrake::Sidekiq::RetryableJobsFilter.new)
```

Comment:
we need this line... this one should not be removed

Reply:
I added back this line and I get

`#17 7.850 NoMethodError: undefined method `child' for #<ActiveSupport::Logger:0x0000564df57ab1d0>`

Reply:
I mean, only this line: `Airbrake.add_filter(Airbrake::Sidekiq::RetryableJobsFilter.new)`, not the one above.